src/client.ts

```diff
@@ -376,7 +376,7 @@ export class Client {
       spanProcessor
     })
 
-    this.instrumentationsLoaded = sdk.start()
```

Comment:
isn't this needed?
https://docs.appsignal.com/nodejs/3.x/instrumentation/instrumentations-load.html#main

Reply:
`sdk.start()` no longer returns a `Promise<void>`, just a good ol' `void`. I'm taking this to mean that the SDK is now loaded synchronously. I need to double-check that assumption before merging.

Before this function is called, the initialiser already sets `client.instrumentationsLoaded()` to `Promise.resolve()`. So `await this.instrumentationsLoaded()` now becomes an unnecessary no-op, but it won't break for users who have added it to their code.

If this is the case and there really is nothing that needs to be awaited on the SDK's initialisation, then we should probably remove the documentation for `instrumentationsLoaded` and deprecate it for removal in v4. We could also put it behind an accessor that emits a warning, telling users that they no longer need to call it.

Reply:
Looked at the change to the SDK start. Here's [the relevant PR](https://github.com/open-telemetry/opentelemetry-js/pull/3460) and [the issue it solved](https://github.com/open-telemetry/opentelemetry-js/issues/2912) -- the issue being the same one we were encountering, which we addressed by exposing the promise as `instrumentationsLoaded`. So the changes in this PR are correct.

I have therefore [created an issue for deprecating `instrumentationsLoaded`.](https://github.com/appsignal/appsignal-nodejs/issues/877)